src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsBaseObject.java

```diff
@@ -601,6 +606,31 @@ public AdsBaseObject triggerAdSlot(String slotName) {
     return this;
   }
 
+  private void triggerFMR() {
+    final By FMRSelector = By.cssSelector(AdsContent.getSlotSelector(AdsContent.FLOATING_MEDREC));
+    scrollToPosition(By.cssSelector("#wikia-recent-activity"));
+
+    doUntilElementVisible(FMRSelector, () -> {
+      jsActions.scrollBy(0, 100);
+      wait.forX(Duration.ofSeconds(1));
+    });
+  }
+
+  private void doUntilElementVisible(By by, Runnable f) {
+    Boolean isElementDisplayed = false;
+    changeImplicitWait(0, TimeUnit.MILLISECONDS);
+
+    do {
+      try {
+        isElementDisplayed = driver.findElement(by).isDisplayed();
+      } catch (NoSuchElementException ignored) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Either log or rethrow this exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1166)